### PR TITLE
:mortar_board: Topic boolean logic --- De Morgan's subsection

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Explain De Morgan's laws

### Why

They are of particular interest as they help with making circuits, also, it's a law that makes it easier to see how `nand` and `nor` are cool. 

### How

I found it tricky to explain the intuition behind it. I feel like the images will help. 

### Testing

:+1:

### CODE REVIEW

File: site/topics/logic/boolean-logic.rst [hunk 1]
```diff
@@ -99,6 +99,155 @@ Properties of Logical Operators
...
+    :math:`\lnot a \lor \lnot b = \lnot(a \land b)`
+
+.. figure:: de_morgans_law_1.png
+    :width: 500 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/De_Morgan%27s_laws
+
+    Visualization of De Morgan's first law. In this figure, blue is what is included in the result, yellow is what is
+    excluded. Here, the union symbol (:math:`\cup`) is equivalent to **or** (:math:`\lor`) and the intersect symbol
+    (:math:`\cap`) is equivalent to **and** (:math:`\land`).
```
Comment (twentylemon) [inline site/topics/logic/boolean-logic.rst]: Does this render ok? The first `)` may be considered part of the `:math:`. Flipping the parens may also make sense, like _Here U (set union) is blah_, and you probably don't need to re-explain logical operators.

File: site/topics/logic/boolean-logic.rst [hunk 2]
```diff
@@ -99,6 +99,155 @@ Properties of Logical Operators
...
+
+* It is simpler to see how the above figure represents :math:`\lnot(a \land b)`
+
+    * Think of the Venn diagram of **and**, then invert it
+
+
+* But also imagine what :math:`\lnot a` and :math:`\lnot b` would be
+
+    * :math:`\lnot a` would be everything *but* what is within :math:`a`, including that which is in :math:`b`
+    * :math:`\lnot b` would be everything *but* what is within :math:`b`, including that which is in :math:`a`
```
Comment (twentylemon) [inline site/topics/logic/boolean-logic.rst:218]: This took a few reads for me, maybe _...including that which is in both a and b_.

File: site/topics/logic/boolean-logic.rst [hunk 3]
```diff
@@ -99,6 +99,155 @@ Properties of Logical Operators
...
+    :width: 500 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/De_Morgan%27s_laws
+
+    Visualization of De Morgan's first law. In this figure, blue is what is included in the result, yellow is what is
+    excluded. Here, the union symbol (:math:`\cup`) is equivalent to **or** (:math:`\lor`) and the intersect symbol
+    (:math:`\cap`) is equivalent to **and** (:math:`\land`).
+
+
+* It is simpler to see how the above figure represents :math:`\lnot(a \land b)`
```
Comment (twentylemon) [inline site/topics/logic/boolean-logic.rst]: _EASIER_

File: site/topics/logic/boolean-logic.rst (hunk 1 above)
Comment (jameshughes89) [reply site/topics/logic/boolean-logic.rst]: It does render OK, but I like your idea of switching the order 

> _Here U (set union)..._

File: site/topics/logic/boolean-logic.rst (hunk 2 above)
Comment (jameshughes89) [reply site/topics/logic/boolean-logic.rst:218]: That doesn't really work here as it's not a but b. 

Idk, I think it makes sense as is.

File: site/topics/logic/boolean-logic.rst (hunk 1 above)
Comment (jameshughes89) [reply site/topics/logic/boolean-logic.rst]: I left the logic there because, whatever.

File: site/topics/logic/boolean-logic.rst (hunk 2 above)
Comment (tmychaskiw-equinix) [reply site/topics/logic/boolean-logic.rst:218]: To be clear, I mean:
> :math:`\lnot a` would be everything *but* what is within :math:`a`, including that which is in both :math:`a` and :math:`b`

**Review by tmychaskiw-equinix — APPROVED** [review]

**Review by twentylemon — APPROVED** [review]
ignore that other idiot's approval, he has no idea what he's doing

**Comment by jameshughes89** [comment]
> ignore that other idiot's approval, he has no idea what he's doing

yeah, That guy is a weirdo

**Pull request merged**